tests/tests/test-balance.ts

```diff
@@ -69,15 +69,16 @@ describeWithMoonbeam("Moonbeam RPC (Balance)", `simple-specs.json`, (context) =>
             events.length === 1 && context.polkadotApi.events.system.ExtrinsicSuccess.is(events[0])
           ).to.be.true;
           break;
-        // Fourth event: ethereum.transact:: system.NewAccount, balances.Endowed, ethereum.Executed,
+        // Fourth event: ethereum.transact:: system.NewAccount, balances.Endowed, (?), ethereum.Executed,
         // system.ExtrinsicSuccess
         case 3:
           expect(section === "ethereum" && method === "transact").to.be.true;
           expect(events.length === 4);
           expect(context.polkadotApi.events.system.NewAccount.is(events[0])).to.be.true;
           expect(context.polkadotApi.events.balances.Endowed.is(events[1])).to.be.true;
-          expect(context.polkadotApi.events.ethereum.Executed.is(events[2])).to.be.true;
-          expect(context.polkadotApi.events.system.ExtrinsicSuccess.is(events[3])).to.be.true;
+          // TODO: what event was inserted here?
+          expect(context.polkadotApi.events.ethereum.Executed.is(events[3])).to.be.true;
```

Comment:
TODO

Reply:
@joelamouche Is this obvious to you, or do you have some documentation handy that you could share?